README.md

```diff
@@ -0,0 +1,31 @@
+# Rails projects authentication
+
+# Things you may want to cover:
+
```

Comment:
- [ ] Try to add a description of the project, what you are trying to achieve and what is it about.